test/cpp/util/cli_credentials.cc

```diff
@@ -22,27 +22,43 @@
 
 DEFINE_bool(enable_ssl, false, "Whether to use ssl/tls.");
 DEFINE_bool(use_auth, false, "Whether to create default google credentials.");
+DEFINE_string(access_token, "",
+              "The access token in metadata \"authorization\".");
 
 namespace grpc {
 namespace testing {
 
 std::shared_ptr<grpc::ChannelCredentials> CliCredentials::GetCredentials()
     const {
-  if (!FLAGS_enable_ssl) {
-    return grpc::InsecureChannelCredentials();
-  } else {
+  if (!FLAGS_access_token.empty()) {
     if (FLAGS_use_auth) {
-      return grpc::GoogleDefaultCredentials();
-    } else {
-      return grpc::SslCredentials(grpc::SslCredentialsOptions());
+      fprintf(
+          stderr,
+          "warning: use_auth is set to false when access_token is provided.");
```

Comment:
Is it intended for `if (!FLAGS_use_auth)` at line 34? Considering the comments, maybe this check can be removed after all.

Reply:
Updated the message. It could be useful to explicitly tell the user that `use_auth` has been surpassed by `access_token` and default google credentials will not be used.